app/controllers/authenticate_controller.rb

```diff
@@ -25,9 +26,25 @@ def status
     )
     render json: { status: "ok" }
   rescue => e
-    render status_failure_response(e)
+    render status_response(e)
   end
 
+  def update
+    body_params = Rack::Utils.parse_nested_query(request.body.read)
+    
+    Authentication::UpdateAuthenticatorConfig.new.(
+      account: params[:account],
+      authenticator: params[:authenticator],
```

Comment:
in the rest of the file, this param is called `authenticator_name`. please align this and update `UpdateAuthenticatorConfig` accordingly